### PR TITLE
cluster: Type -> Rule in confirmTopology

### PR DESCRIPTION
**Pull request by 9547**
### What problem does this PR solve? 

IN `tiup cluster display` the topology's header is like below:

image

But in `tiup cluster scale-out` the topology's header is like below:

image

Let's be consistent

### Check List 

Tests 

 - Unit test
 - Integration test
 - Manual test (add detailed scripts or steps below)
 - No code

Code changes

 - Has exported function/method change
 - Has exported variable/fields change
 - Has interface methods change
 - Has persistent data change

Side effects

 - Possible performance regression
 - Increased code complexity
 - Breaking backward compatibility

Related changes

 - Need to cherry-pick to the release branch
 - Need to update the documentation

Release notes:

```release-note
NONE
```

### CODE REVIEW

File: pkg/cluster/manager/manager.go [hunk 1]
```diff
@@ -87,7 +87,7 @@ func (m *Manager) confirmTopology(name, version string, topo spec.Topology, patc
 
 	clusterTable := [][]string{
 		// Header
-		{"Type", "Host", "Ports", "OS/Arch", "Directories"},
+		{"Rule", "Host", "Ports", "OS/Arch", "Directories"},
```
Comment (lucklove) [inline pkg/cluster/manager/manager.go:90]: Maybe `Role`?

Comment (9547) [reply pkg/cluster/manager/manager.go:90]: Sorry :sweat_smile:

**Pull request merged**